publish/rocky/8/rocky_linux_8.wf.json

```diff
@@ -1,8 +1,8 @@
 {
   "Name": "rocky-linux-8",
-  "Project": "gce-image-builder",
+  "Project": "gce-ciq-images",
   "Zone": "us-central1-b",
```

Comment:
I'd suggest being mindful of the zone that the build is run it, specifically if they all use the same zone/region then you may hit quota issues or need to request more.

Small tweaks here of using different regions for Rocky 8 and 9 images means you reduce any quota risk. This same method can be used to help have less quota issues for CIT.